acmw-db-app/package.json

```diff
@@ -1,17 +1,30 @@
 {
```

Comment:
This file should not be changed at all, i.e. it should not even appear here in the file changed tab